### ISSUE TITLE
Add fallback to e2e test execution

### ISSUE THREAD

**Issue by dguendisch** [gardener/test-infra#341]
We should have some fallback when encountering a cluster with e.g. k8s 1.21 but no desc files are available yet (i.e. I just added some via https://github.com/gardener/test-infra/pull/340).
The fallback could just take the most recent desc files for the given clusters k8s version.

@schrodit wdyt?

**Comment by schrodit** [comment]
Yes I think it makes sense as we also only do a copy paste of the last version.
Although it was a good indicator that we have to adapt the configuration. otherwise we might forget about it🤔

**Comment by dguendisch** [comment]
> Although it was a good indicator that we have to adapt the configuration. otherwise we might forget about it🤔

Actually we don't adapt them but still just copy them (otherwise we'd break the release train), so I don't think this should be "misused" as a reminder :)

**Comment by schrodit** [comment]
> Actually we don't adapt them but still just copy them (otherwise we'd break the release train)
yes valid point. Then lets default to the latest available version
